mgmtd/mgmt_txn.c

```diff
@@ -2385,6 +2386,55 @@ int mgmt_txn_send_get_tree_oper(uint64_t txn_id, uint64_t req_id,
 	get_tree->simple_xpath = simple_xpath;
 	get_tree->xpath = XSTRDUP(MTYPE_MGMTD_XPATH, xpath);
 
+	if (CHECK_FLAG(flags, GET_DATA_FLAG_CONFIG)) {
+		struct mgmt_ds_ctx *ds =
+			mgmt_ds_get_ctx_by_id(mm, MGMTD_DS_RUNNING);
+		struct nb_config *config = mgmt_ds_get_nb_config(ds);
+
+		if (config) {
+			struct ly_set *set = NULL;
+			LY_ERR err;
+
+			err = lyd_find_xpath(config->dnode, xpath, &set);
+			if (err) {
```

Comment:
`set` is leaked (never `ly_set_free(set, NULL)`d)

Reply:
Thanks, fixed.